apps/studio/src/pages/sites/[siteId]/pages/[pageId]/index.tsx

```diff
@@ -69,7 +70,12 @@ function EditPage(): JSX.Element {
         </EditorDrawerProvider>
       </TabPanel>
       <TabPanel>
-        <PageSettings type={type} permalink={permalink} title={title} />
+        <PageSettings
+          type={type}
+          permalink={permalink.split("/").at(-1) || "/"}
+          prefix={permalink.split("/").slice(0, -1).join("/")}
```

Comment:
should the trpc procedure return the `relativePermalink` and `prefix` instead of doing this frontend conversion? if not might desync.